development/process.md

```diff
@@ -102,12 +101,21 @@ keep the amount of rework from the contributor to a minimum.
       website are updated.
     * Finally the release is announced.
 
+## Code style
+
+All contributions must follow the project's code style guidelines to ensure
+consistency and readability. For Java code, follow the [Trino Java code style guide](https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#code-style).
+For SQL and documentation, refer to the respective sections in the [development guide](https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md).
```

Comment:
Where is the style about SQL in this page?

Reply:
Did you want to link to https://github.com/trinodb/trino/blob/master/docs/README.md instead?

Reply:
Removed SQL part and left code style only